pkg/kubelet/rkt/rkt.go

```diff
@@ -142,6 +142,8 @@ const (
 
 	etcHostsPath      = "/etc/hosts"
 	etcResolvConfPath = "/etc/resolv.conf"
+
+	restartDone = "done"
```

Comment:
I think this const needs a comment to clearly indicate that it represents a result of calling a dbus method.

As-is, it's too easy to miss that this can't just be an arbitrary string.

In an ideal world, I think this would actually be exported by the go-systemd package, but for now just having a comment saying what / why seems necessary.

Also, this isn't strictly just a "resartDone" value, this is actually just indicating that the dbus operation is done, and this value is also used for stop, not just restart.